.env.defaults

```diff
@@ -1,5 +1,5 @@
 COIN_NETWORK=mainnet
-SITE_URL=https://coin.space/api/v1/
+SITE_URL=https://coin.space/api/
```

Comment:
Add to [./server/.env.example](https://github.com/CoinSpace/CoinSpace/blob/develop/server/.env.example) also.